### PR TITLE
test: fix window undefined issue for tests

### PR DESCRIPTION
**Pull request by antsukanova**
# COMPLETES # [SPARK-555432](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-555432)

## This pull request addresses

PR addresses issues where test cases run with `yarn workspace @webex/plugin-meetings test:unit ` that command works well until some new folders were added or old folders were removed and an issue with _"window is not defined"_ appeared.

Here is also [the task](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-566398) for further investigation, but this PR just fixes an issue with _"window is not defined"_ issue for existing tests cases that fail when run the command `yarn workspace @webex/plugin-meetings test:unit --targets "filename"`

## by making the following changes

- add missed imports for stubbing window object
- remove unused variables

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [x] Internal code refactor

## The following scenarios were tested

- builds sdk successfully

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ This is for compliance purposes with FedRAMP program.
 - [ ] Tooling change
 - [ ] Internal code refactor
 
-## The following scenarios where tested
+## The following scenarios were tested
```
Comment (antsukanova) [inline .github/PULL_REQUEST_TEMPLATE.md:35]: I feel it's time to update this

File: packages/@webex/plugin-meetings/test/unit/spec/personal-meeting-room/personal-meeting-room.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /*!
  * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
  */
-
+import 'jsdom-global/register';
 import 'jsdom-global/register';
```
Comment (rarajes2) [inline packages/@webex/plugin-meetings/test/unit/spec/personal-meeting-room/personal-meeting-room.js:5]: Added redundantly.

**Review by rarajes2 — COMMENTED** [review]
Is there no other way than adding `jsdom-global`?

File: packages/@webex/plugin-meetings/test/unit/spec/personal-meeting-room/personal-meeting-room.js (hunk 2 above)
Comment (antsukanova) [reply packages/@webex/plugin-meetings/test/unit/spec/personal-meeting-room/personal-meeting-room.js:5]: @rarajes2 thanks for comment, will update

**Comment by antsukanova** [comment]
@rarajes2 answering your comment about another way, I have tried to fix it by adding a new flag "--jsdom" for `yarn workspace @webex/plugin-meetings test:unit `, but I've stuck with some jsdom object work mismatch (jsdom in the global Mocha config and in file), and I'm not sure that adding a rule to all files makes sense, it should be investigated separately. Also, I noticed that `yarn workspace @webex/plugin-meetings test:unit` works strangely because it skips some errors that are shown when you run only one file for a check. Even with undefined, ideally, I'd expect to see this issue with `yarn workspace @webex/plugin-meetings test:unit`. I added this as a separate task to SDK team to investigate this [the task](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-566398).

**Review by edvujic — APPROVED** [review]
LGTM, thanks :)

**Review by rarajes2 — APPROVED** [review]

**Review by brycetham — APPROVED** [review]

File: packages/@webex/plugin-meetings/test/integration/spec/converged-space-meetings.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import { config } from 'dotenv';
 import 'jsdom-global/register';
+import { config } from 'dotenv';
```
Comment (sreenara) [inline packages/@webex/plugin-meetings/test/integration/spec/converged-space-meetings.js:2]: can we fix the formatting here?

**Review by sreenara — APPROVED** [review]
@antsukanova please fix the formatting error in the import. Everything else is ok for now.

File: packages/@webex/plugin-meetings/test/integration/spec/converged-space-meetings.js (hunk 3 above)
Comment (antsukanova) [reply packages/@webex/plugin-meetings/test/integration/spec/converged-space-meetings.js:2]: Wonder why it wasn't fixed automatically.. 
updated, thank you

**Pull request merged**